src/pybind/mgr/dashboard/plugins/ses7p_motd.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import hashlib
+import json
+
+from mgr_module import Option
+
+from . import PLUGIN_MANAGER as PM
+from . import interfaces as I  # noqa: E741,N812
+
+
+@PM.add_plugin  # pylint: disable=too-many-ancestors
+class SES7PMotd(I.CanMgr, I.Setupable, I.HasOptions):
+    @PM.add_hook
+    def get_options(self):
+        return [Option(
+            name='ses7p_motd_configured',
+            default=False,
+            type='bool',
+            desc='Is the MOTD for SES7P configured')]
+
+    @PM.add_hook
+    def setup(self):
+        # Check whether the MOTD has already been configured. This is
+        # done to ensure the SES7P announcement is not configured on
+        # every start of the Ceph Dashboard module and to allow the
+        # administrator to disable it if necessary.
+        configured = self.mgr.get_module_option('ses7p_motd_configured')
+        if configured:
+            return
+        # Note, we can not call the 'ceph dashboard motd set ...' command
+        # here because it is not available at this time. To workaround
+        # this issue we build and write the MOTD configuration ourself.
+        message = 'There is an upgrade to SES7.1 available, please check the ' \
+                  '<a href="https://documentation.suse.com/ses/7/html/ses-all/ses-upgrade.html">documentation</a> ' \
```

Comment:
@tbazant Is the URL https://documentation.suse.com/ses/7/html/ses-all/ses-upgrade.html still correct?

Reply:
this URL ^^^ referes to both upgrade paths (SES 6 -> 7.1 and SES 7 -> 7.1). To refer to 7 -> 7.1 only, use https://documentation.suse.com/ses/7/html/ses-all/upgrade-to-pacific.html (does not exist yet but should be correct).

Reply:
> this URL ^^^ referes to both upgrade paths (SES 6 -> 7.1 and SES 7 -> 7.1). To refer to 7 -> 7.1 only, use [documentation.suse.com/ses/7/html/ses-all/upgrade-to-pacific.html](https://documentation.suse.com/ses/7/html/ses-all/upgrade-to-pacific.html) (does not exist yet but should be correct).

OK, i will adapt the URL to ^^^^^ because the MOTD refers to SES7->SES7.1.